test/unit/sync_mediator_test.rb

```diff
@@ -189,4 +195,10 @@ def sync(mediator)
       mediator.sync([]) # clears the cache
     end
   end
+
+  class FakeHPA < MockResource
+    def kubectl_resource_type
+      'fakeHPA'
```

Comment:
Isn't this the same as the value of `type`?  If so the test isn't proving anything

Reply:
[1] pry(#<SyncMediatorTest>)> hpa.kubectl_resource_type
=> "fakeHPA"
[2] pry(#<SyncMediatorTest>)> hpa.type
=> "FakeHPA"

I'll update to make it more clear.